### ISSUE TITLE
Hardcoded settings

### ISSUE THREAD

**Issue by FinGu** [FinGu/c_auth#7]
DB and captcha settings are hardcoded.

The db settings are stored in functions/general/settings.php

The google captcha settings in : 
https://github.com/FinGu/c_auth/blob/main/register.php#L5,
https://github.com/FinGu/c_auth/blob/main/register.php#L143
https://github.com/FinGu/c_auth/blob/main/reset_password.php#L9
https://github.com/FinGu/c_auth/blob/main/reset_password.php#L134

That's a bad practice as far as i know ( atleast for compiled languages ) that should be avoided.
I won't 'fix' it since i have no motivation to do so.

**Comment by Drknowi** [comment]
and the Capcha Secret?

**Comment by wnelson03** [comment]
And he changed it so it's easier to see where the secret is supposed to go now 👍

**Comment by JustinWrld** [comment]
Do you have a c# form example and not Console giving me headaches

**Comment by wnelson03** [comment]
Here's an example of a Winform example close to cAuth https://github.com/KeyAuth/KeyAuth-CSHARP-Example/tree/main/WinformExample. Fingu gave me permission to use client-side code from his Project. Only hard part for most people is adding serialization reference, just right click references, type "serialization" and add that reference. Everything else should be quite self explanatory.

**Comment by FinGu** [comment]
I didn’t, the code is free to use

**Comment by wnelson03** [comment]
Yeah sorry I said it wrong. You helped me understand the crypto because that was the one part I had trouble understanding from the outside. Appreciate it <3. By the way, not sure if you're interested in working on your uploader repository, though you don't need to write file before encrypting it, I'm using the code `$contents = file_get_contents($_FILES['file_to']['tmp_name']);` because I switched from OVH to Krystal.uk due to OVH's firewall being terrible, causing website API to be inaccessible, and terrible dashboard, plus no email hosting, and shit support. Anyways, after I made the switch to Krystal.uk, I had to encrypt the file contents because anti-virus. At first, I wrote to file then encrypted, like your repository, though the anti-virus still picked that up and sent me numerous warnings to remove malware from server or risk termination. I've been using the above code which doesn't write to file before encrypting, and I've had no issues for weeks now. Just wanted to say thank you Fingu for your all your help.
![image](https://user-images.githubusercontent.com/83034852/120130574-84aa3580-c194-11eb-88ce-69c58882df17.png)
KeyAuth made this in the last 24 hours. You were an extreme motivation to make my own auth, I had wanted to for a year and you taking the time to explain the crypto pushed me towards my goal tremendously. My PHP code is definitely not as neat and advanced as your code is, which is why I'm not going to attempt to create a pull request on your file repository, because I may not be understanding how the array you feed into the function works, and you may not even care to update it, since the repository works and it's just a free example for others to modify themselves.

Hope you're doing well at whatever you're doing now. I saw you posted a repository programmed in Rust. One of my administrators made an example for KeyAuth in Rust, he's a cool guy. Anyways, take care. Just wanted to say thanks and it seems I got my chance 👍

**Comment by FinGu** [comment]
No problem.
